src/ansiblelint/yaml_utils.py

```diff
@@ -817,8 +819,12 @@ def __init__(
                 - name: Task
         """
         # Default to reading/dumping YAML 1.1 (ruamel.yaml defaults to 1.2)
```

Comment:
This comment is out of date now, because this uses ruamel.yaml's default of 1.2. It should probably be moved to `transformer.py`.